roles/reset/tasks/main.yml

```diff
@@ -29,20 +29,44 @@
   loop_control:
     loop_var: mounted_fs
 
+#
+# it may become or be a common practice to separate /var/lib/rancher and /var/lib/kubelet into their own volumes.  Maybe it would be better to just move /var altogether, but that's not
+# terribly convenient for remote or headless machines (e.g. raspberry pi compute modules, embedded in Turing Pi or Desk Pi where you can't get to single user mode.
+# Recommendation, in this case, is to mount a new volume, such as /k3s-data or similar and symlink /var/lib/rancher and /var/lib/kubelet into directories 
+# other k3s data points like /run/k3s are small/transient/in memory and shouldn't be on a volume
+# 
+# The approach is to delete the files in the folders first, then remove the folders/files that are not symlinks in the follow up
+#
+- name: find service files within the directories that can by symlinks, binaries and data for people who have separate volume and symlink for /var/lib/rancher and /var/lib/kubelet
+  ansible.builtin.find:
+    paths:
+      - /var/lib/rancher/
```

Comment:
This doesn't seem to be where k3s places its files unless you are doing something custom, which we cannot support custom configurations